sycl/source/esimd_emulator_device_interface.cpp

```diff
@@ -62,13 +64,14 @@ __SYCL_EXPORT ESIMDDeviceInterface *getESIMDDeviceInterface() {
 
   // Now check that device interface version is compatible.
   if (Interface->version < ESIMD_DEVICE_INTERFACE_VERSION) {
-    std::cerr << __FUNCTION__ << std::endl
-              << "The device interface version provided from plug-in "
-              << "library is behind required device interface version"
-              << std::endl
-              << "Found version : " << Interface->version << std::endl
-              << "Required version :" << ESIMD_DEVICE_INTERFACE_VERSION
-              << std::endl;
+    fprintf(stderr,
+            "%s\n"
+            "The device interface version provided from plug-in"
+            "library is behind required device interface version\n"
+            "Found version : %llu\n"
```

Comment:
%llu does not seem to be right for printing uintptr_t. See e.g. https://stackoverflow.com/questions/5795978/string-format-for-intptr-t-and-uintptr-t